tests/framework/installer/ceph_manifests.go

```diff
@@ -1992,28 +1949,11 @@ spec:
           secretKeyRef:
             name: rook-ceph-mon
             key: admin-secret
-    securityContext:
```

Comment:
We need the toolbox to be privileged and have the mounts so that the flex test will pass. Reverting these changes to the toolbox should fix it.